docs/design-survey-conditional-logic.md

```diff
@@ -692,4 +692,60 @@ const surveyJson = {
 
 If the `expression` is `true`, the trigger runs another expression specified by the `runExpression` property. You can also save the result of `runExpression` as a question value. For this, assign the question's name to the `setToName` property.
 
-[View the "Run Expression Trigger" example](https://surveyjs.io/Examples/Library?id=trigger-runexpression (linkStyle))
\ No newline at end of file
+[View the "Run Expression Trigger" example](https://surveyjs.io/Examples/Library?id=trigger-runexpression (linkStyle))
+
+---
+
+### `skip`
+
+Jumps to the target question's page and activates the question when `expression` evaluates to true. The `gotoName` property specifies the target question.
+
+In the following code, a trigger navigates to the `"additionalInfoPage"` page and activates the `"additionalInfo"` question if the `"sameAsBilling"` question is `true`:
+
+```js
+const surveyJson = {
+ "pages": [
+  {
+   "name": "billingAddressPage",
+   "elements": [
+    {     
+     "name": "billingAddress",
+     // ...     
+    }
+   ]
+  },
+  {
+   "name": "shippingAddressPage",
+   "elements": [
+    {
+     "type": "boolean",
+     "name": "sameAsBilling"     
+    },
+    {     
+     "name": "shippingAddress",
+     "visibleIf": "{sameAsBilling} = false",
+     // ...
+    }
+   ]
+  },
+  {
+   "name": "additionalInfoPage",
+   "elements": [
+    {     
+     "name": "additionalInfo",
+     // ...
+    }
+   ]
+  }
+ ],
+ "triggers": [
+  {
+   "type": "skip",
+   "expression": "{sameAsBilling} = true",
+   "gotoName": "additionalInfo"
+  }
+ ]
+}
```

Comment:
Let's keep the same configuration for the `sameAsBilling` question so as not to confuse the reader. Also, it would be better if the code sample occupied fewer lines.

```suggestion
const surveyJson = {
 "pages": [{
   "name": "billingAddressPage",
   "elements": [{
     "name": "billingAddress",
     // ...
    }]
  }, {
   "name": "shippingAddressPage",
   "elements": [{
      "name": "sameAsBilling",
      "choices": [ "Yes", "No" ]
      // ...
  }, {     
     "name": "shippingAddress",
     "visibleIf": "{sameAsBilling} = 'No'",
     // ...
    }]
  }, {
   "name": "additionalInfoPage",
   "elements": [{
     "name": "additionalInfo",
     // ...
    }]
  }],
 "triggers": [{
   "type": "skip",
   "expression": "{sameAsBilling} = 'Yes'",
   "gotoName": "additionalInfo"
  }]
}
```